epsagon/trace.py

```diff
@@ -685,23 +685,26 @@ def verify_custom_label(self, key, value):
         :param value: Value for the label data (string)
         :return: True/False
         """
-        if not isinstance(key, str) or not isinstance(value, str):
-            print('EPSAGON: epsagon.label() only supports '
-                  'label keys and values of type string.')
+        if not isinstance(key, str):
+            print('EPSAGON: label key support only string type')
+            print('Received {key}:{value}'.format(key=key, value=value))
+            return False
+        if not isinstance(value, (int, float, str)):
+            print('EPSAGON: label value support only string, int, float types')
             print('Received {key}:{value}'.format(key=key, value=value))
             return False
 
-        if len(key) + len(value) > MAX_LABEL_SIZE:
+        if len(key) + len(str(value)) > MAX_LABEL_SIZE:
```

Comment:
I think it's correct but tricky - only because we do a JSON dump of the labels (of the whole trace actually).
A comment will be nice.

Reply:
👍